.teamcity/settings.kts

```diff
@@ -201,7 +204,11 @@ class TestBuild(val os: String, val arch: String, version: String, buildId: Abso
                     scriptMode = file {
                         path = "_scripts/test_windows.ps1"
                     }
-                    param("jetbrains_powershell_scriptArguments", "-version ${"go$version"} -arch $arch")
+                    if (arch == "arm64") {
+                        param("jetbrains_powershell_scriptArguments", "-version ${"go$version"} -arch amd64 -winarm64")
```

Comment:
I don't understand how this change works. You are making test_windows.ps1 download the amd64 version of the toolchain and then run the tests by setting GOARCH manually to arm64? Why?

Reply:
We still don't have a windows arm64 builder to run tests, but at least we can cross-compile from the windows amd64 builder to check for compilation errors.

Reply:
But is this going to work given that there are no arm64 agents? And when there will be (if there will be any) it definitely won't work because they will try to execute the amd64 version of the go toolchain. I'd just add a build command at the end of test_windows.ps1.

Reply:
I'll add the build command at the end of the ps1 script.